rpm/gssdp.spec

```diff
@@ -11,7 +11,7 @@ BuildRequires: pkgconfig
 BuildRequires: meson
 BuildRequires: vala
 BuildRequires: pkgconfig(glib-2.0)
-BuildRequires: pkgconfig(libsoup-2.4)
+BuildRequires: pkgconfig(libsoup-3.0)
```

Comment:
We don't have this. Since you have built this, you probably have it - maybe just forgotten PR for libsoup update?

Reply:
The plan was to have a separate package for libsoup3 so all don't have to be ported to it right now.